Packs/CVESearchV2/Integrations/CVESearch/CVESearch.py

```diff
@@ -0,0 +1,128 @@
+import urllib3
+from typing import Dict, Any, Tuple, List
+
+from CommonServerPython import *
+
+# Disable insecure warnings
+urllib3.disable_warnings()
+
+
+class Client(BaseClient):
+    """
+    Client will implement the service API, and should not contain any Demisto logic.
+    Should only do requests and return data.
+    """
+
+    def __init__(self, base_url: str):
+        headers = {
+            'Content-Type': 'application/json',
+            'Accept': 'application/json'
+        }
+        super().__init__(base_url=base_url, headers=headers)
+
+    def http_request(self, suffix: str):
+        """Connects to api and Returns response.
+           Args:
+               suffix :The API endpoint.
+           Returns:
+               response from the api.
+           """
+        return self._http_request(method='GET', url_suffix=suffix)
+
+    def cve_latest(self) -> List[Dict[str, Any]]:
+        res: List[Dict[str, Any]] = self.http_request(suffix='/last')  # type: ignore
+        return res
+
+    def cve(self, cve_id) -> Dict[str, Any]:
+        res: Dict[str, Any] = self.http_request(f'cve/{cve_id}')
+        if not res:
+            return_error(f'Could not find cve results for ID {cve_id}')
```

Comment:
this is problematic
if the issue is with invalid cve id provided, so we should verify the id before sending it